src/styles/_utility.scss

```diff
@@ -105,16 +105,17 @@ $line-heights-map: (
   ),
 );
 
+// TODO: rename to $font-size-scales-map
 $scales-map: (
   //TODO: rename qpc_uthmani_hafs to text_qpc_hafs
   qpc_uthmani_hafs:
     (
       tablet: (
         1: 3.2vh,
         2: 3.5vh,
-        3: 4vh,
-        4: 4vh,
-        5: 4.4vh,
+        3: 3.6vh,
+        4: 3.7vh,
+        5: 3.8vh,
```

Comment:
These changes should reflect also in `$line-heights-map` and `$skeleton-height-map`:
1. `$line-heights-map` to make sure the entire viewport is filled for 1 page for the reading view.
2.   `$skeleton-height-map` to avoid having a mismatch between the page height when it's showing the skeleton and when it's actually showing the Quranic text.